packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -79,18 +80,32 @@ export class FileBrowserService implements ServiceMethods<any> {
   async get(directory: string, params?: UserParams): Promise<Paginated<FileContentType>> {
     if (!params) params = {}
     if (!params.query) params.query = {}
-    const { $skip, $limit, storageProviderName } = params.query
+    const { $recursive, $skip, $limit, storageProviderName } = params.query
 
     delete params.query.storageProviderName
     const skip = $skip ? $skip : 0
     const limit = $limit ? $limit : 100
+    const recursive = $recursive
```

Comment:
feathers pagination properties begin with $, we dont' (and shouldnt) follow this convention for other APIs. this can just be `recursive`

Reply:
Got it, little experience with feathers so I was just following the pattern I saw.